docs/guide/user_management/user_management.md

```diff
@@ -134,6 +134,20 @@ It will check all User accounts in the database and list those that do not fit t
 
 You can allow your users to create accounts by employing the `/register` route. This route leads to a registration form that, when filled in, creates a new User Content item in the repository.
 
+### User types
+
+To decide to what workspace the user should be registered to, you need to specify their user type under `user_type_identifier`.
```

Comment:
We don't use "workspace" anywhere else, either the term needs explanation, or rephrase.